plugins/TodayAtMun/month.py

```diff
@@ -0,0 +1,14 @@
+month = {
```

Comment:
you can probably generate this data for you instead of writing it out by hand, see:
https://stackoverflow.com/questions/3418050/month-name-to-month-number-and-vice-versa-in-python

Reply:
Completed. Easy fix